packages/react-storage/src/components/StorageBrowser/Views/Controls/Paginate.tsx

```diff
@@ -31,90 +71,197 @@ const iconAttributes = {
   xmlns: 'http://www.w3.org/2000/svg',
 };
 
-const nextIconProps = () => ({
-  children: (
-    <path
-      d="M9.99984 6L8.58984 7.41L13.1698 12L8.58984 16.59L9.99984 18L15.9998 12L9.99984 6Z"
-      fill="currentColor"
-    ></path>
-  ),
-  ...iconAttributes,
-});
+const getIconProps = ({
```

Comment:
Is this only scoped to the paginate component? Would it make sense to define it somewhere else so it could be reused with other icons?

Reply:
That's the plan